server/server.go

```diff
@@ -66,24 +67,62 @@ type config struct {
 	Port             string `envconfig:"PORT" default:"6789"`
 	ReportPath       string `envconfig:"REPORT_PATH" default:""`
 	PathPrefix       string `envconfig:"KEPLOY_PATH_PREFIX" default:"/"`
+
+	// Use this option to tune the behaviour of the path where the logs would be written.
+	LogLocation models.LogLocation `envconfig:"LOG_LOCATION" default:"console"`
+
+	// The absolute or relative filepath of the files to store the logs in.
+	// This is platform agnostic, so forward and backward slashes both are fine.
+	// If the file doesn't exists, one will be created for you.
+	LogFileLocation string `envconfig:"LOG_FILE_LOCATION" default:"/tmp/keploy.log"`
 }
 
 func Server(ver string) *chi.Mux {
 	rand.Seed(time.Now().UTC().UnixNano())
 
+	// Parse the config before creating the logger to figure out whether we need
+	// to pipe the logs or not.
+	var conf config
+	err := envconfig.Process("keploy", &conf)
+	if err != nil {
+		fmt.Printf("failed to read/process configuration with error %v", err)
+		return nil
+	}
+
+	// Convert the separators in the filepath to a platform agnostic version.
+	// For example, if user passed in "/C/Downloads", it'd be converted to
+	// "\C\Downloads" in Windows.
+	conf.LogFileLocation = filepath.FromSlash(conf.LogFileLocation)
+
+	// If we need to pipe or redirect the logs, create the relevant directories.
+	if conf.LogLocation != models.Console {
+		err := CreateNestedDirectoriesIfNotExists(conf.LogFileLocation)
+		if err != nil {
+			fmt.Println("Could not create nested directories for filepath due to error %v", err)
```

Comment:
Please use log.Fatal to log error

Reply:
The reason for using `fmt.Print` instead of `logger.Print` was that the logger wasn't created by that point.

Sure, we can temporarily use inbuilt `log` package while our logger is initialized. Updated.